linkage/pvs_like_case_study/README.md

```diff
@@ -36,8 +38,36 @@ $ Rscript -e "renv::restore(library=.libPaths())"
 Updating R packages should be done within the environment, installing them
 and then calling `renv::snapshot(type = "all")`.
 
+### Local Spark version
+
+Unfortunately, it isn't possible to install Spark with conda.
+Instead, I have used a Singularity image with Spark, and then activated
+a conda environment inside it.
+
+These instructions basically only work on the IHME cluster, because they assume that the
+location of your conda, and the location where it creates new conda environments,
+both are subdirectories of `/mnt`.
+Also, the singularity pull assumes amd64 architecture.
+
+```
+$ conda create -n pvs_like_case_study_spark_local --file=pvs_like_case_study_spark_local_lock_no_jupyter.txt # or if you need jupyter, leave out the no_jupyter
+$ singularity pull spark.sif docker://apache/spark@sha256:a1dd2487a97fb5e35c5a5b409e830b501a92919029c62f9a559b13c4f5c50f63
```

Comment:
What's the origin of this docker path?

Reply:
`singularity pull docker://...` uses Docker Hub. This is [the official Spark image on Docker Hub](https://hub.docker.com/r/apache/spark), and I pinned the exact version I used to run this.